cypress/e2e/A11y/customer_details.cy.js

```diff
@@ -0,0 +1,36 @@
+import 'cypress-axe'
+
+describe('Detalhes dos clientes - Verifica se há problemas de acessibilidade', () => {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.get('tbody tr').find('td').eq(1).click()
+        cy.injectAxe()
+    })
+
+    it('Não encontra problemas de acessibilidade no modo claro', () => {
+        cy.get('.customer-details').checkA11y()
```

Comment:
Aqui deveria ser o seguinte:

```suggestion
        cy.checkA11y()
```

Reply:
O mesmo vale para outros testes de _a11y_.